internal/gqlparser/rewrite-deps.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Relies on GNU sed to do in-place regex search-and-replace for the module strings.
```

Comment:
Sed, GNU Sed, it's often a hassle when running stuff both on linux and macos. (Like, brew installs `gsed`, not `sed`)... I guess we could do something that came up in this thread, https://groups.google.com/g/golang-nuts/c/RyhFfhB_kd8, like

1. a couple of `gofmt -w -r` calls, one per package that's got to be rewritten
2. try https://github.com/rogpeppe/govers.

[In another script, I ended up using perl because it seemed the only cross-platform way to replace a string inplace 😅 ](https://sourcegraph.com/github.com/open-policy-agent/opa/-/blob/build/update-version.sh?L4-6&subtree=true)

Reply:
I switched it over to Perl, because that allowed me to avoid having to rewrite the search/replace regexes. The `rewrite-deps.sh` script now relies on Perl for its simple string replacements.